docs/modules/integrations/pages/azure-devops-exporter.adoc

```diff
@@ -8,15 +8,17 @@ Features:
 
 == Azure DevOps Export Properties
 
-[cols="2,1,3", options="header"]
+[cols="2,1,1,3", options="header"]
 |===
 
 |Property
```

Comment:
@valfirst  does it make sense to follow common rules for properties definition across all the docs and made required props as a bold

Reply:
yes, it makes sense